### PR TITLE
Properties

### PR DESCRIPTION
**Pull request by fomentia**
Work tracked in #39.

### CODE REVIEW

File: src/converters/entities/OSXConfigurationEntitiesConverter.test.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      idleTime: 600,
+      loginWindowModulePath: "/System/Library/Screen Savers/Flurry.saver",
+    },
+  ],
+};
+
+const expectedEntity: OSXConfigurationEntity = {
+  _class: "Configuration",
+  _key: "jamf_osx_configuration_profile_1",
+  _type: "jamf_osx_configuration_profile",
```
Comment (aiwilliams) [inline src/converters/entities/OSXConfigurationEntitiesConverter.test.ts]: There'll need to be some code that deletes the deprecated type this replaces.

File: src/jupiterone/relationships/AccountOSXConfigurationRelationship.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { RelationshipFromIntegration } from "@jupiterone/jupiter-managed-integration-sdk";
+
+export type AccountOSXConfigurationRelationship = RelationshipFromIntegration;
+
+export const ACCOUNT_OSX_CONFIGURATION_RELATIONSHIP_TYPE =
+  "jamf_account_has_jamf_osx_configuration_profile";
```
Comment (aiwilliams) [inline src/jupiterone/relationships/AccountOSXConfigurationRelationship.ts]: I don't think we need to have `jamf` in here twice. I'd prefer `jamf_account_has_osx_configuration_profile`. No biggie though, the type values on relationships I hope are never used in user-facing stuff 🙏

File: src/types.ts [hunk 3]
```diff
@@ -18,6 +18,15 @@ export interface Account {
...
 }
 
+export interface Site {
+  id: number;
+  name: string;
+}
+
+export type Category = Site;
+
+export type Plist = string;
```
Comment (aiwilliams) [inline src/types.ts]: Kinda wish this were `PListString` or `PListDocument`.

File: src/converters/entities/OSXConfigurationEntitiesConverter.test.ts (hunk 1 above)
Comment (fomentia) [reply src/converters/entities/OSXConfigurationEntitiesConverter.test.ts]: In place now

File: src/jupiterone/relationships/AccountOSXConfigurationRelationship.ts (hunk 2 above)
Comment (fomentia) [reply src/jupiterone/relationships/AccountOSXConfigurationRelationship.ts]: yeah, I agree. This is the way they have all of them, though, and I didn't want to update all of the relationship types right now.

**Pull request merged**